src/main/java/org/hbase/async/HBaseClient.java

```diff
@@ -985,23 +985,12 @@ public Scanner newScanner(final String table) {
     if (LOG.isDebugEnabled()) {
       LOG.debug("BigTable API: Scanning table with {}", scanner.toString());
     }
-    Table table = null;
-    try {
-      table = hbase_connection.getTable(TableName.valueOf(scanner.table()));
-      ResultScanner result = table.getScanner(scanner.getHbaseScan());
-      scanner.setResultScanner(result);
-      scanner.setHbaseTable(table);
-
-      return Deferred.fromResult(new Object());
-    } catch (IOException e) {
-      if (table != null) {
-        try {
-          table.close();
-        } catch (Exception e1) {}
-      }
 
-      return Deferred.fromError(e);
-    }
+    AsyncTable table = hbase_asyncConnection.getTable(TableName.valueOf(scanner.table()), executor);
+    ResultScanner result = table.getScanner(scanner.getHbaseScan());
+    scanner.setResultScanner(result);
+    scanner.setHbaseTable(table);
```

Comment:
Does this setHbaseTable actually do anything?  If not, can we remove the method?